x-pack/test_serverless/api_integration/test_suites/common/index_management/settings.ts

```diff
@@ -8,33 +8,32 @@
 import expect from '@kbn/expect';
 
 import { FtrProviderContext } from '../../../ftr_provider_context';
+import { RoleCredentials } from '../../../../shared/services';
 
 export default function ({ getService }: FtrProviderContext) {
-  const indexManagementService = getService('indexManagement');
-
-  describe('settings', () => {
-    let getIndexSettings: typeof indexManagementService['settings']['api']['getIndexSettings'];
-    let updateIndexSettings: typeof indexManagementService['settings']['api']['updateIndexSettings'];
-    let createIndex: typeof indexManagementService['indices']['helpers']['createIndex'];
-    let deleteAllIndices: typeof indexManagementService['indices']['helpers']['deleteAllIndices'];
+  const svlCommonApi = getService('svlCommonApi');
+  const svlUserManager = getService('svlUserManager');
+  const svlSettingsApi = getService('svlSettingsApi');
+  const svlIndicesHelpers = getService('svlIndicesHelpers');
+  let roleAuthc: RoleCredentials;
 
+  describe('settings', function () {
+    // see details: https://github.com/elastic/kibana/issues/187369
+    this.tags(['skipSvlOblt', 'skipSvlSearch', 'skipSvlSec']);
     before(async () => {
-      ({
-        settings: {
-          api: { getIndexSettings, updateIndexSettings },
-        },
-        indices: {
-          helpers: { createIndex, deleteAllIndices },
-        },
-      } = indexManagementService);
+      roleAuthc = await svlUserManager.createApiKeyForRole('admin');
     });
 
-    after(async () => await deleteAllIndices());
+    after(async () => {
+      await svlIndicesHelpers.deleteAllIndices();
+      await svlUserManager.invalidateApiKeyForRole(roleAuthc);
+    });
 
-    it('should fetch an index settings', async () => {
-      const index = await createIndex();
+    xit('should fetch an index settings', async () => {
```

Comment:
Same as my comment above about `xit`. Also, the whole suite is skipped anyway, so might not be needed anymore?

Reply:
https://github.com/elastic/kibana/pull/184105/commits/ae3caed5ea1578ec7e73e7b92a509b905d31e34a